### DISCUSSION TITLE
How to DIY the input/output format

### DISCUSSION THREAD

**Discussion by ruifengma** [BerriAI/litellm#4677] [Ideas]
If I would like to modify a bit the input/output structure for the api service, what should I do? Thanks

**Comment by ghost** [comment]
Moving to discussion, as this isn't exposed yet - but will try and enable it, this week

**Reply by ruifengma** [reply]
Thanks @krrishdholakia for the reply, I have a very similar api request input format to openai api and I would like to adapt. Now I can connect to my local hosted LLMs with the config file and I'm find the definition about the format. Or if it is possible to get some information about where did you define it this

**Reply by ruifengma** [reply]
> Moving to discussion, as this isn't exposed yet - but will try and enable it, this week

any progress?

**Reply by ghost** [reply]
yup - working on it now @ruifengma

**Reply by ghost** [reply]
PR - https://github.com/BerriAI/litellm/pull/4887
